libnavigation-base/src/main/java/com/mapbox/navigation/base/road/model/RoadComponent.kt

```diff
@@ -4,12 +4,14 @@ import com.mapbox.api.directions.v5.models.MapboxShield
 
 /**
  * Object that holds road components
- * @property text of the road
+ * @property text name of the road. May be '/' if used for separation of road names
```

Comment:
I think we need more information and examples about the separator usage. Could you add it to the docs?

Reply:
@abhishek1508 @averkhaturau had a conversation about it. Could you please help with a docs update and examples?

Reply:
I'd suggest to remove separators with '/' text, just because there is no logic in them. Currently just every even road name is '/', and it was done for back-compatibility when the road name field was just textual. Users can add a separator themselves  if needed, even with any other type of separator. Let's remove them?

Reply:
@S2Ler , I mean road names containing just a slash. We add "/" road names as a legacy and I don't know why. So I'd suggest to not add them.

Reply:
Nav SDK at present doesn't have enough guidance on how to use `/`. We use it the same way we use it to display banner instructions. 
@averkhaturau Since NN, is still sending `/` to support backwards compatibility we would still be using `/` to separate road names in cases multiple of them exist for a current road name or else we are gonna break the behavior for our customers. Customers who don't want to use `/` can still filter them out on their end.

We can change the documentation as follows

```
@property text contains the current road name user is on, based on the [language] available. In certain situations there can be [RoadComponent] that contain [text] as `/` separator. The purpose of these separators is to separate the current road name in cases they have a primary and secondary name. If you are using [MapboxRoadNameView], then these separators are used to separate these names. However if you don't want to use them, you are can directly get access to [RoadComponent] from [LocationMatcherResult] and filter the names without `/` separators.
```  

cc @korshaknn @LukasPaczos

Reply:
@abhishek1508 , but if NavSDK will filter road names by languages, some '/' may remain in wrong place. Note that '/' has empty language tag, and also will present between road names in different languages.
As currently we provide road names as a collection, user can add separators when it's needed. Initially '/' was used to separate different road names when they were provided as a String.
But I'm generally ok to leave separators as it. I just doubt that they are needed and will not lead to errors due to code complication when some names will be filtered out based on language.

Reply:
Sure, that's what I suggest is to remove slashes in NN. For this I need NavSDK approval.

Reply:
We decided to leave slashes as is and will consider improving the interface next versions

Reply:
We just had a call and the decision is to keep the `/` so that both iOS and Android SDK keep working as they are. Also both iOS and Android don't need to do any filtering on the UI side we don't have a defined requirement.

Resolving the ticket.